bootstrap_cfn/fab_tasks.py

```diff
@@ -478,6 +491,12 @@ def _validate_fabric_env():
     elif not os.path.isfile(env.config):
         sys.exit("\n[ERROR] Config file %s does not exist" % str(env.config))
 
+    # keyname is mandatory in cfn_create, optional in others.
+    # if _validate_fabric_env() is called by cfn_create, then check keyname.
+    # otherwise not.
+    if called_by_cfn_create and env.keyname is None:
+            sys.exit("\n[ERROR] Please specify a keyname, e.g 'keyname:keyops'")
+
```

Comment:
Maybe accept it if its in the config, but warning that it's deprecated/not-advised. whatever
Only a suggestion to think about though, it's not a requirement. If you're sure this is the best way then fire on